core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/message.properties

```diff
@@ -0,0 +1,3 @@
+warningMessage=There are users that are still using a legacy API token. \
```

Comment:
It would be a little nicer to change "who" -> "that".

See [Grammar Girl](https://www.quickanddirtytips.com/education/grammar/who-versus-that)

Reply:
So I think it's `that` -> `who` that you meant ?